test_files/lobster/vasp_lobster_output/FW_submit.script

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I don't think this file should be in here.

Reply:
will remove this.